rllib/evaluation/worker_set.py

```diff
@@ -268,7 +399,8 @@ def add_policy(
                 Callable[[PolicyID, Optional[SampleBatchType]], bool],
             ]
         ] = None,
-        workers: Optional[List[Union[RolloutWorker, ActorHandle]]] = None,
+        # Deprecated.
+        workers: Optional[List[Union[RolloutWorker, ActorHandle]]] = DEPRECATED_VALUE,
```

Comment:
Same comment as above: Users may want to add policies to only certain remote workers.
I know this would make things much more complicated, but let's at least think this through first.

Reply:
😄